docs/get-started.md

```diff
@@ -7,29 +7,41 @@ This tutorial provides a quick introduction to use `Nebula Graph`.
 
 ### Step 1 Install Nebula Graph
 
-The easiest way to startup `nebula` is using `docker`.
+The easiest way to startup `Nebula` is using `Docker`.
 `Docker` is a tool designed to make it easier to create, deploy, and run applications by using containers.
 Containers allow a developer to package up an application with all of the parts it needs,
 such as libraries and other dependencies, and ship it all out as one package.
 By doing so, the developer can rest assured that the application will run on any other `Linux` machine regardless of any customized settings that machine might have that could differ from the machine used for writing and testing the code.
 
-First of all, you should make sure that `docker` has been installed on your machine. Open a terminal and run the following command :
+First of all, you should make sure that `Docker` has been installed on your machine. Open a terminal and run the following command :
 
 ```
 docker --version
 ```
 
-If `docker` is not found, please see [here](https://docs.docker.com/install/) for more information to install docker.
+If `Docker` is not found, please see [here](https://docs.docker.com/install/) for more information to install Docker.
 
 After that, using
 ```
 docker pull vesoft/nebula-graph:latest
 ```
-to get `nebula` image and `docker images` that can display images status.
+to get `Nebula` image and `Docker images` that can display images status.
 
-If `docker` is slow when pulling the image, configure an accelerator.
+If `Docker` is slow when pulling the image, configure a new mirror.
```

Comment:
Ditto.

Besides, what is slow is the very pulling action itself, but not Docker.